src/tools/index.md

```diff
@@ -130,6 +130,10 @@ Here are the general-purpose tools you might want to install:
 [dart_style]: {{site.pub-pkg}}/dart_style
 [dartfix]: {{site.pub-pkg}}/dartfix
 
+### Debugging and diagnostics
```

Comment:
Should this be under **Tools for developing command-line apps and servers**?

Reply:
I think debugging is one of the main gaps this addresses for console apps.